hqscripts/management/commands/makemigrations.py

```diff
@@ -71,8 +71,8 @@ def assert_exclusive_options(opt_name, extra=[]):
             with patch_migration_autodetector(self):
                 super().handle(*app_labels, **options)
 
-    def write_migration_files(self, changes):
-        super().write_migration_files(changes)
+    def write_migration_files(self, *args):
+        super().write_migration_files(*args)
```

Comment:
Thanks for catching this and sorry for missing it!

Reply:
I think the error only happened when `makemigrations` was run with `--update`, so the issue probably was not obvious when running without that option.